precode.sh

```diff
@@ -6,38 +6,56 @@
 #   dir2
 #   dir3
 #       dir4
+mkdir -p task/dir1
```

Comment:
Команду можно объединить 
```suggestion
mkdir -p task/dir1 task/dir2 task/dir3/dir4
```